src/style-dictionary-dist/TerrawareTheme.ts

```diff
@@ -1,3 +1,5 @@
+import { PaletteOptions } from '@mui/material/styles/createPalette';
```

Comment:
Is this necessary ?

Reply:
oh i ran this design workflows change github action, didn't make this change manually..
probably need to fix the action since it was written a long time ago